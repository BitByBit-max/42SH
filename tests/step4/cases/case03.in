#!/bin/bash

talk()
{
    case "${name}" in
        "Elise")
            echo "hello elise"
            ;;
        "Enrique")
            echo "wake up enrique"
            ;;
        "Elsa")
            echo "is it done yet?"
            ;;
    esac
}

name="Elsa"
talk
name="Enrique"
talk
name="Elise"
talk
