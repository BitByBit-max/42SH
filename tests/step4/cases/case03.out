is it done yet?
wake up enrique
hello elise
